### DISCUSSION TITLE
文档官网

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#2907] [分享 / Share]
确实有看到这个目录，~~但这东西从TP一直到现在都没有更新，所以我就花了点时间搭了个文档~~
![image](https://user-images.githubusercontent.com/47922588/150287720-4e96999c-3bd2-4938-8118-3007dfaae13c.png)

#### 文档的来历
最近在写 React，~~又放假了~~，便用 Next.js 写了一个官网，看下大家喜不喜欢（
[网站地址](https://bilibili-evolved-doc.vercel.app) , [项目地址](https://github.com/FoundTheWOUT/bilibili-evolved-doc)

#### PS
现在还算是比较粗糙的版本，有些东西还不是很完善
另外，如果可以，我也愿意把项目 transfer 到 bv 下

**Comment by the1812** [comment]
> 确实有看到这个目录，~~但这东西从TP一直到现在都没有更新，所以我就花了点时间搭了个文档~~

这个只是预留而已, 并没有任何开发计划(

**Comment by the1812** [comment]
项目的话就还放你那好了, 等完善了我在 README 这边放个链接.

以及... 我好像没把这个写到文档来着, ~~这居然被你发现了~~ 它和 `debugger` 类似, 参数的单位是毫秒不是秒
![image](https://user-images.githubusercontent.com/26504152/150469781-841a46d7-2842-44f6-b30c-a3ae98f4ad85.png)

**Reply by FoundTheWOUT** [reply]
现在基本稳定了，搜索也已经做好了(‾◡◝)
